diff -Naurp neutron-10.0.0.0rc1.orig/neutron/tests/etc/neutron.conf neutron-10.0.0.0rc1/neutron/tests/etc/neutron.conf
--- neutron-10.0.0.0rc1.orig/neutron/tests/etc/neutron.conf	2017-02-03 20:04:08.000000000 -0500
+++ neutron-10.0.0.0rc1/neutron/tests/etc/neutron.conf	2017-02-06 13:40:43.835688795 -0500
@@ -23,5 +23,5 @@ rpc_backend = fake
 lock_path = $state_path/lock
 
 [database]
-connection = 'sqlite://'
+connection = 'sqlite://sqlite.db'
 
