--- a/etc/neutron.conf
+++ b/etc/neutron.conf
@@ -87,7 +87,7 @@
 #web_framework = legacy
 
 # The core plugin Neutron will use (string value)
-#core_plugin = <None>
+core_plugin = ml2
 
 # Range of seconds to randomly delay when starting the periodic task scheduler
 # to reduce stampeding. (Disable by setting to 0) (integer value)
@@ -594,7 +594,7 @@
 # /etc/neutron/rootwrap.conf' to use the real root filter facility. Change to
 # 'sudo' to skip the filtering and just run the command directly. (string
 # value)
-#root_helper = sudo
+root_helper = sudo /usr/bin/neutron-rootwrap /etc/neutron/rootwrap.conf
 
 # Root helper daemon application to use when possible. (string value)
 #root_helper_daemon = <None>
@@ -683,7 +683,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection = <None>
+connection = sqlite:////var/lib/neutron/neutron.sqlite
 
 # If True, SQLite uses synchronous mode. (boolean value)
 # Deprecated group/name - [DEFAULT]/sqlite_synchronous
